Phrases.java

```diff
@@ -1,232 +1,98 @@
 package ChatBot;
+import com.google.gson.Gson;
+import com.google.gson.JsonParser;
+import java.io.FileReader;
+import java.io.BufferedReader;
 
 class Phrases {
-  static final String[] wasDoingSomething = {
-    "blinked",
-    "chocked",
-    "closed his eyes",
-    "saw a pretty person far away",
-    "shed a tear",
-    "sneezed",
-    "was coughing",
-    "was feeling dizzy",
-    "was feeling tired",
-    "was having a silly smile",
-    "was hesitating",
-    "was itching his <censored>",
-    "was itching his navel",
-    "was picking his teeth",
-    "was scratching his back",
-    "was staring at the ceiling",
-    "was thinking about justice in the world",
-    "was too self confident",
-    "was trying to get rid of a fly",
-    "was trying to make a hit",
-    "was waiting for a hit"
-  };
 
-  static final String[] when = {
-    "and at the same time",
-    "and that's when",
-    "just as",
-    "meanwhile",
-    "when out of nowhere",
-    "when suddenly",
-    "when",
-    "whereas"
-  };
+  static String[] wasDoingSomething;
+  static String[] when;
+  static String[] adjective;
+  static String[] wasTrying;
+  static String[] but;
+  static String[] blocked;
+  static String[] missed;
+  static String[] said;
+  static String[] wisdomIntro;
+  static String[] wisdoms;
+  static String[] won;
+  static String[] lookingForOpponent;
+
+  static String[][] hit;
+  static String[][] toHit;
 
-  static final String[] adjective = {
-    "furious",
-    "dangerous",
-    "rabid",
-    "mad",
-    "wild",
-    "frenzied",
-    "impudent",
-    "insolent",
-    "shameless",
-    "bold",
-    "insidious",
-    "treacherous",
-    "cunning",
-    "sly",
-    "crafty",
-    "naughty",
-    "wicked",
-    "rude",
-    "nasty",
-    "timid",
-    "shy"
+  static String getLang() {
```

Comment:
as you mentioned, this is not in the right place. Maybe add TODO to move it later. Suggested TODO format:
```
// TODO(Riboloff): ask the user which language they prefer. Move this to appropriate class.
```